### DISCUSSION TITLE
Cannot deploy Admin on Netflify and / or NextJS

### DISCUSSION THREAD

**Discussion by fxmb** [medusajs/medusa#2775] [Q&A] [answered]
- We tried pretty much anything, deployed it both on netlify and vercel. 
- We removed the `dist` folder as it does not exist and the deployment fails otherwise. 
- After deployment of the admin UI, the URL is just unresponsive. I checked the network tab, console. etc. - just nothing.
- It is not CORS related, it does not event connect to the server even though the correct URL is provided.
- Locally, the admin server does not start again once stopped once, it just throws a `Access to localhost was denied` once we stop and restart the server locally....

Any pointers on how to best deploy the Admin board? We feel like the Docs are often times lacking information or even carry wrong information (e.g. the `dist` folder)... Would be super helpful to get admin up and running...

**Comment by fxmb** [comment, marked as answer]
We found the issue: 
- The build out put is NOT in the `dist` folder as put by netflify and vercel as default, but should be changed to `public` as can be seen by the `vite.config.ts`. It is working now, would be helpful to add this to the docs.

**Reply by srindom** [reply]
Thanks for reporting @fxmb. We recently migrated to Vite -- sorry for not having documented this previously!

cc: @shahednasser

**Reply by shahednasser** [reply]
We have a PR open for this, should be updated soon 😊

**Reply by fxmb** [reply]
Great, thanks for being so responsive
